app/docker-compose.yml

```diff
@@ -3,10 +3,10 @@ services:
     database:
         image: mysql:8.0.31
         ports:
-            - "3306:3306"
+            - "3306:5432"
         environment:
             MYSQL_ROOT_PASSWORD: password
-            MYSQL_DATABASE: valtimo-implementation-dev
+            POSTGRES_DB: valtimo-implementation-dev
```

Comment:
Is this some unfortunate replace-all result? Because it's weird to set postgres related variables to a mysql docker.